cli/src/config/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod build;
```

Comment:
@flyin I'm not happy with the current approach to public parts of the crate, but will discuss it later.